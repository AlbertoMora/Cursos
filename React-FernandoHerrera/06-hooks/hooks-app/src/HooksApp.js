import React from 'react'

const HooksApp = () => {
    return (
        <div>
            <h1>React Hooks</h1>
        </div>
    )
}

export default HooksApp;