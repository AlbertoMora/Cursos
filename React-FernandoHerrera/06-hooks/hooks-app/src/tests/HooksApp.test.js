import React from "react";
import { shallow } from "enzyme";

import HooksApp from "../HooksApp";

describe("Test over HooksApp functionality", () => {
    test("should match snapshot", () => {
        const wrapper = shallow(<HooksApp />);
        expect(wrapper).toMatchSnapshot();
    });
});
