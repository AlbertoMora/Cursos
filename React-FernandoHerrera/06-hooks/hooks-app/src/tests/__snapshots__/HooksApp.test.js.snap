// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test over HooksApp functionality should match snapshot 1`] = `
<div>
  <h1>
    React Hooks
  </h1>
</div>
`;
